test/JDBC/expected/database_metadata.out

```diff
@@ -47,14 +47,14 @@ master#!#dbo#!#dbmeta_view1#!#VIEW#!#<NULL>
 
 dbmeta#!#getColumns#!#master|dbo|dbmeta_tab1
 ~~START~~
-varchar#!#varchar#!#varchar#!#varchar#!#smallint#!#varchar#!#int#!#int#!#smallint#!#smallint#!#smallint#!#varchar#!#nvarchar#!#smallint#!#smallint#!#int#!#int#!#varchar#!#int#!#int#!#int#!#tinyint#!#text#!#text#!#smallint#!#smallint#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
+varchar#!#varchar#!#varchar#!#varchar#!#smallint#!#varchar#!#int#!#int#!#smallint#!#smallint#!#smallint#!#varchar#!#nvarchar#!#smallint#!#smallint#!#int#!#int#!#varchar#!#int#!#int#!#int#!#tinyint#!#varchar#!#varchar#!#smallint#!#smallint#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
```

Comment:
how is the fix for case expression result type is affecting here ?

Reply:
After running this JDBC tests, I have generated the queries used in "dbmeta#!#getColumns#!#master|dbo|dbmeta_tab1" inside logfile. It is seen that it is implemented using CASE statement. 
logfile:
"
sp_executesql: handle: -1 statement: select [master].sys.schemas.name 'TABLE_SCHEM', CASE WHEN [master].sys.schemas.name  IN  ('dbo', 'guest','INFORMATION_SCHEMA','sys','db_owner', 'db_accessadmin', 'db_securityadmin', 'db_ddladmin'  ,'db_backupoperator','db_datareader','db_datawriter','db_denydatareader','db_denydatawriter')  THEN null ELSE 'master'  END 'TABLE_CATALOG'    from [master].sys.schemas where [master].sys.schemas.name like @P0   order by 2, 1"
"
sql_batch statement: select [master].sys.schemas.name 'TABLE_SCHEM', CASE WHEN [master].sys.schemas.name  IN  ('dbo', 'guest','INFORMATION_SCHEMA','sys','db_owner', 'db_accessadmin', 'db_securityadmin', 'db_ddladmin'  ,'db_backupoperator','db_datareader','db_datawriter','db_denydatareader','db_denydatawriter')  THEN null ELSE 'master'  END 'TABLE_CATALOG'    from [master].sys.schemas order by 2, 1"